packages/buckets/src/api/index.ts

```diff
@@ -656,13 +685,18 @@ export function bucketsPullPath(
           opts.progress(written)
         }
       },
-      onEnd: async (status: grpc.Code, message: string, _trailers: grpc.Metadata) => {
+      onEnd: async (
+        status: grpc.Code,
+        message: string,
+        // _trailers: grpc.Metadata,
+      ) => {
         if (status !== grpc.Code.OK) {
           fail(new Error(message))
         }
         stop()
       },
     })
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
This is related to the above new linter setting. We don't always need to be strict if we have a well-defined response like this.